dist/api_data.py

```diff
@@ -983,6 +983,14 @@ def __ge__(self, other):
         size and the default config would extend log files in allocations of
         the maximum log file size.''',
         type='list', choices=['data', 'log']),
+    Config('file_close_sync', 'true', r'''
+        control whether to flush modified tables to storage independent
```

Comment:
Overall, I think we need something reworded because the only time this setting can be effective is if logging is disabled. I think it should start with that prominently. Maybe something like:
"when false, and logging is disabled, API calls that require exclusive access to tables will return EBUSY if there have been changes made to the table since the last global checkpoint. When true, or when logging is enabled, modified tables will be flushed to storage when closing table when required for APIs needing exclusive access."

Reply:
Ah this is what I started with, somehow changed my mind and reworded it in the current form. I'll fix this.